pages/VerifiableCredentials/Types/VerifiableCredentialSchema.md

```diff
@@ -0,0 +1,182 @@
+# Verifiable Credential Schema
+
+The schema for a DSNP Verifiable Credential MAY be defined using the format described in the [Verifiable Credentials JSON Schema](https://www.w3.org/TR/vc-json-schema/) W3C Candidate Recommendation Draft.
+This format provides a credential wrapper around a JSON schema document.
+
+Empty schemas (Verifiable Credential Schema Documents with `"jsonSchema": {}`) are allowed; however, schemaless credentials may be preferred in this situation.
+Empty schemas are useful in situations where no attribute data fields are relevant but the schema author wishes to assert authorship.
+
+## Verifiable Credential Schema Document
+
+A Verifiable Credential Schema document is a JSON-LD document that is itself a [Verifiable Credential](./VerifiableCredential.md) with a claim that includes the schema and (optionally) the DSNP extension described below.
+A schema credential specializes the meaning of the following fields:
+
+| Property | Required | JSON Type | Description | Restrictions |
+| --- | --- | --- | --- | --- |
+| `type` | YES | Array of strings | Type of credential | MUST contain the strings `"VerifiableCredential"` and `"JsonSchemaCredential"` |
+|`credentialSubject` | YES | Object | Object containing JSON schema and DSNP extensions | See [Credential Subject](#credential-subject) |
+| `credentialSchema` | YES | Object | Metaschema defining JSON Schema types | See [Credential Schema](#credential-schema) |
+
+### Credential Schema
+The required `credentialSchema` object MUST follow the specification and contain:
+
+```
+{
+  "id": "https://www.w3.org/2022/credentials/v2/json-schema-credential-schema.json",
+  "type": "JsonSchema",
+  "digestSRI": "sha384-S57yQDg1MTzF56Oi9DbSQ14u7jBy0RDdx0YbeV7shwhCS88G8SCXeFq82PafhCrW"
+}
+```
+
+### Credential Subject
+
+A DSNP Verifiable Credential Schema document's `credentialSubject` object uses the following keys:
+
+| Property | Required | JSON Type | Description | Restrictions |
+| --- | --- | --- | --- | --- |
+| `type` | YES | String | Type of subject matter | MUST be `JsonSchema` |
+| `jsonSchema` | YES | Object | Embedded JSON Schema object | See [JSON Schema](#json-schema) |
+| `dsnp` | NO | Object | DSNP extension object | See [DSNP Extensions](#dsnp-extensions) |
```

Comment:
Again YES/NO is YES/no in other parts of the document.  And this link does not work.